### DISCUSSION TITLE
Poll: How many people are still using BepInEx 6?

### DISCUSSION THREAD

**Discussion by slyh** [slyh/Cities2-TrafficLightsEnhancement#10] [Polls]
I’m considering dropping support for BepInEx 6 if the usage is low.

**Comment by PsykotropyK** [comment]
Why should v6 have a lower usage than v5 ? Does it have any drawbacks ?

**Reply by slyh** [reply]
Not that it has any drawbacks in terms of functionality. It's just that BepInEx 6 is still in pre-release state and a lot of mods are now targeting BepInEx 5.

**Reply by PsykotropyK** [reply]
With BepInEx 5 the mod as not functionnal on my machine. Dunno why, though I only tested with the first version of the mod (which I found on modscities2.com), so it may anyway work now.

**Reply by Karnor** [reply]
I'm not a maintainer but as a user one drawback of 6 is that currently 6 does not work on a linux system, so linux users of which I am not the only one, would be cut off from much needed mods like the traffic light manager and pedestrian pathfinding.

**Reply by slyh** [reply]
I think BepInEx 6 can patch at runtime on Linux as well.
https://steamcommunity.com/sharedfiles/filedetails/?id=2841673787

**Comment by HarryBotter2333** [comment]
There are still many mods under development that only support BepInEx Version 5. I can only give up using V6 for now.

**Comment by snottyone** [comment]
the available "mod manager" only gives you bepinex 5
which is among the reasons why i ditched that quickly.

**Comment by Gtasian** [comment]
I was using bepinex6 only because this mod required it at the beginning. Now I'm using 5 with the mod manager.

**Comment by Zaltocleotl** [comment]
I only joined github to comment how much I love the mod!
I started on 5, but this required 6 and it was THE mod I wanted at the time, so I switched to 6. All same day. I hadn't modded my game until this mod appeared. I'm still on 6. I should check and see if going back to five would cut out the other two mods I consider mandatory right now, or even worth going to for anything I don't have right now.
THANKS AGAIN! The new alpha is much improved again. It's working fine and looks great. I was wondering if the lane markers for lane control would get improved. Sure did!
